config/default.yml

```diff
@@ -304,6 +304,7 @@ Style/HashSyntax:
   SupportedStyles:
     - ruby19
     - hash_rockets
+    - dont_mix
```

Comment:
Maybe:

* unmixed
* consistent
* uniform

None of those are great... I'll keep thinking about it. Better to get that
right before merging and being stuck with it.